RELEASE.markdown

```diff
@@ -0,0 +1,26 @@
+# How to Release
+
+## Create a stable release
+
+1. Create a BLADE jira ticket for the new release
+2. Create a new branch based on ticket
+3. Run `gradlew prepareRelease`
+4. Commit the changes
+5. Generate release notes (@lawrence-lee to provide this command)
```

Comment:
@lawrence-lee  can you provide these details?

Reply:
After the changes are committed, from `liferay-blade-cli` run `./gradlew -b changelogs-sdk.gradle buildChangelogs`

Reply:
You should see these two generated files. Commit both of them.

```> Task :recordCommits
Updated '/opt/dev/projects/github/liferay-blade-cli/CHANGELOG.commits'

> Task :buildChangelogs
Updated '/opt/dev/projects/github/liferay-blade-cli/CHANGELOG.markdown'```